code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -19,6 +19,11 @@
 	pass_flags_self = PASSSTRUCTURE | LETPASSTHROW
 	var/crate_climb_time = 20
 	var/obj/item/paper/fluff/jobs/cargo/manifest/manifest
+	/// Gives a crate a lid sprite if it has one. Leave Blank if it doesn't.
+	var/lid_icon = 'icons/obj/storage/crates.dmi'
+	var/lid_icon_state
+	var/lid_x = 0 //X moves the lid left or right.
```

Comment:
Please make this into a doc comment like the one on `lid_icon`, and make sure all of your new vars have one.